app/presenters/report_presenter.rb

```diff
@@ -16,6 +16,18 @@ def deadline
     I18n.l(super)
   end
 
+  def approved_at
+    return if super.blank?
+    I18n.l(super, {format: :detailed})
+  end
+
+  def uploaded_at
+    return nil if export_filename.nil?
+    timestamp = Time.parse(export_filename[-18..-5])
```

Comment:
This is an instance of `Time` parsed from a timestamp, so the variable name could be clearer

Reply:
Have tried to make this clearer